collections/date-time/calculate-the-number-of-months-between-two-dates.md

```diff
@@ -0,0 +1,6 @@
+~~~ javascript
+const monthDiff = (startDate, endDate)  => (endDate.getFullYear() - startDate.getFullYear()) * 12 - startDate.getMonth() + endDate.getMonth() <= 0 ? 0 : (endDate.getFullYear() - startDate.getFullYear()) * 12 - startDate.getMonth() + endDate.getMonth()
```

Comment:
Should it be
`Math.max(0, (endDate.getFullYear() - startDate.getFullYear()) * 12 - startDate.getMonth() + endDate.getMonth())`?

Reply:
Yeah nice that's much cleaner. I'll make a new commit.